django/home/serializers.py

```diff
@@ -112,6 +113,32 @@ class MemberProfileSerializer(serializers.ModelSerializer):
     bio = MarkdownField()
     research_interests = MarkdownField()
 
+    def validate_affiliations(self, value):
```

Comment:
this is good, should we also include a Django model validator? https://docs.djangoproject.com/en/4.1/ref/validators/